x/pocketcore/types/aat.go

```diff
@@ -66,7 +67,8 @@ func (a AAT) Bytes() []byte {
 		Version:              a.Version,
 	})
 	if err != nil {
-		panic(fmt.Sprintf("an error occured hashing the aat:\n%v", err))
+		fmt.Println(fmt.Sprintf("an error occured hashing the aat:\n%v", err))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("an error occured hashing the aat:\n%v", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)